src/main/kotlin/no/nav/omsorgspenger/overføringer/Vurderinger.kt

```diff
@@ -71,9 +74,29 @@ internal object Vurderinger {
                 }
             }
 
+        val borSammenMed = relasjoner.filter { it.borSammen }.map { it.identitetsnummer }
+        val barnSomBorNånAnnenstans = alleBarn
+            .filterNot { it.identitetsnummer in borSammenMed }
+
+        alleBarn.forEach { barn ->
+            behandling.lovanvendelser.leggTil(
+                periode = behandling.periode,
+                lovhenvisning = OmsorgenForBarnet,
+                anvendelse = when (barn in barnSomBorNånAnnenstans) {
+                    true -> "Bor ikke sammen med barnet født ${barn.fødselsdato}"
+                    false -> "Bor sammen med barnet født ${barn.fødselsdato}"
+                }
+            )
+        }
+
+        if(grunnlag.overføreOmsorgsdager.overførerTil !in borSammenMed) {
+            behandling.leggTilKarakteristikk(Behandling.Karakteristikk.OppfyllerIkkeInngangsvilkår)
```

Comment:
Skall vi ha en egen karakteristikk før dette?

Reply:
Tror vi kan bruke denne, men legg også til lovanvendelse for EktefelleEllerSamboer